README.md

```diff
@@ -56,5 +56,96 @@ console.log(errors.AccessDenied);
 
 ```
 
+## Clustering
+
+The clustering class can be used to set up a cluster of workers. The class will
+create at least 1 worker, will log any worker event (started, exited).
+The class has also a watchdog who will restart workers until the stop() method
+is called.
+
+### Usage
+
+#### Simple
+
+```
+import { Clustering } from 'arsenal';
+
+const clusters = new Clustering(clusterSize, logger);
```

Comment:
Since we only get one cluster, let the variable be singular as well.
`const cluster`